README.txt

```diff
@@ -10,4 +10,8 @@
 
 4) BE CAREFUL. Modifications through use of the application may create unintended database modifications
 
-5) The database is NOT LOCAL. Cloud-based database, any modifications on the database must be done with caution.
\ No newline at end of file
+5) The database is NOT LOCAL. Cloud-based database, any modifications on the database must be done with caution.
+
+6) A Firebase Service Account Key is REQUIED to run the program and can be given if requested due to Google Privacy Commitment Policy. The JSON file could not be uploaded.
```

Comment:
We could insert a valid key into the bundle before submitting it instead of this